### DISCUSSION TITLE
skywalking9.7.0 using ES storage,no data from 00:00 to 00:04 every day

### DISCUSSION THREAD

**Discussion by waningmoon-stack** [apache/skywalking#12232] [Q&A] [answered]
core:
enableDataKeeperExecutor:true
dataKeeperExecutePeriod:5
recordDataTTL:15
metricsDataTTL:15
persistentPeriod:25
elasticsearch:
dayStep:1

**Comment by wu-sheng** [comment]
It may cause by your elastic performance. In that moment, indices are auto created by server when data flows in.

**Reply by waningmoon-stack** [reply, marked as answer]
thx for your repay, I will check es

**Reply by waningmoon-stack** [reply]
Excuse me, I have another question to ask you.
When storing data in Elasticsearch, can I create an index manually in advance?
Create an index for tomorrow in advance, for example, at 11:00 PM tonight?

**Reply by wu-sheng** [reply]
> When storing data in Elasticsearch, can I create an index manually in advance?

If you write a script to do so, yes, you can.

**Reply by waningmoon-stack** [reply]
thanks
